userland/AppMakefile.mk

```diff
@@ -66,19 +100,19 @@ $$(BUILDDIR)/$(1):
 
 # First step doesn't actually compile, just generate header dependency information
 # More info on our approach here: http://stackoverflow.com/questions/97338
-$$(BUILDDIR)/$(1)/%.o: %.c | $$(BUILDDIR)/$(1)
+$$(BUILDDIR)/$(1)/%.o: %.c | $$(BUILDDIR)/$(1) $$(HDRS)
```

Comment:
I expected them to have their directories `-I` included, but I don't see where that's happening. My concern was just that it seemed to be a requirement that we were then not using.

Is the idea that if there was some process for generating the headers, this would cause that to run?